### PR TITLE
Chore(access): replace deprecated CONFIG

### PR DESCRIPTION
**Pull request by bruno-infotrad**
Replace deprecated CONFIG with elgg_get_config()

### CODE REVIEW

File: engine/lib/access.php [hunk 1]
```diff
@@ -116,13 +116,12 @@ function get_access_array($user_guid = 0, $site_guid = 0, $flush = false) {
...
 function get_default_access(ElggUser $user = null, array $input_params = array()) {
-	global $CONFIG;
 
 	// site default access
-	$default_access = $CONFIG->default_access;
+	$default_access = elgg_get_config('default_access');
 
 	// user default access if enabled
-	if ($CONFIG->allow_user_default_access) {
+	if (elgg_get_config('allow_user_default_access') {
```
Comment (mrclay) [inline engine/lib/access.php]: Missing closing parenthesis. It's recommended you use an IDE with syntax checking.

**Comment by mrclay** [comment]
Commit message should start with "chore" (not capitalized).

**Comment by jdalsem** [comment]
Please resubmit this against 2.x if you want this in

**Comment by mrclay** [comment]
I wouldn't bother with `$CONFIG` changes in private API.

**Pull request closed without merging**